src/Boehm-GC.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="Boehm-GC"
+   name="Boehm-Demers-Weiser GC License" listVersionAdded="3.21">
+      <crossRefs>
+         <crossRef>https://fedoraproject.org/wiki/Licensing:MIT#Another_Minimal_variant_(found_in_libatomic_ops)</crossRef>
+         <crossRef>https://github.com/uim/libgcroots/blob/master/COPYING</crossRef>
+         <crossRef>https://github.com/ivmai/libatomic_ops/blob/master/LICENSE</crossRef>
+      </crossRefs>
+      <text>
+         <p>
+            THIS MATERIAL IS PROVIDED AS IS, WITH ABSOLUTELY NO
+            WARRANTY EXPRESSED OR IMPLIED. ANY USE IS AT YOUR OWN RISK.
+         </p>
+         <p>
+            Permission is hereby granted to use or copy this program for any
+            purpose, provided the above notices are retained on all copies.
+            Permission to modify the code and to distribute modified code is
+            granted, provided the above notices are retained, and a notice that
+            the code was modified is included with the above copyright notice.
+         </p>
+         <p>
+            A few files have other copyright holders.
+         </p>

```

Comment:
@jlovejoy I would suggest either omitting lines 22-24 or else making them optional. Neither the Fedora wiki nor libatomic_ops have it. If it's here and not optional then it won't be a match...